desktop/core/src/desktop/js/apps/storageBrowser/StorageBrowserPage/StorageBrowserActions/StorageBrowserActions.test.tsx

```diff
@@ -56,56 +56,124 @@ describe('StorageBrowserRowActions', () => {
     }
   ];
 
-  test('does not render view summary option when there are multiple records', async () => {
-    const user = userEvent.setup();
-    const { getByRole, queryByRole } = render(
-      <StorageBrowserActions selectedFiles={mockRecord2} />
-    );
-    await user.click(getByRole('button'));
-    expect(queryByRole('menuitem', { name: 'View Summary' })).toBeNull();
-  });
+  const setLoadingFiles = jest.fn();
+  const setRefreshKey = jest.fn();
+  describe('Summary option', () => {
```

Comment:
For the scenarios when we do render the actions, why only check for the menu item not being null? I would test that I can actually click it that something is happening.